addons/purchase/tests/test_purchase_invoice.py

```diff
@@ -502,6 +502,8 @@ def test_partial_billing_interaction_with_invoicing_switch_threshold(self):
 
         self.assertEqual(line.qty_invoiced, 10)
 
+        invoice.invalidate_cache(fnames=['payment_state', 'state'])
```

Comment:
shouldn't this be below the invoicing_switch_threshold? if I understood this was what was changing the value so it makes more sense functionally (even if it should work if we don't access the field values inbetween)

Reply:
Oops, you're right, it should be below the invoicing_switch_threshold.
I'll fix it, thanks !